Week 04/Lecture 08/Assignment 02/employee_manager/src/main/java/com/example/employee_manager/dao/EmployeeDAO.java

```diff
@@ -0,0 +1,62 @@
+package com.example.employee_manager.dao;
+
+import com.example.employee_manager.model.Employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Repository
+public class EmployeeDAO {
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    // SQL queries
+    private final String INSERT_QUERY = "INSERT INTO Employee (ID, Name, Address, Department) VALUES (?, ?, ?, ?)";
+    private final String SELECT_QUERY = "SELECT * FROM Employee";
+    private final String SELECT_BY_ID_QUERY = "SELECT * FROM Employee WHERE ID = ?";
+    private final String UPDATE_QUERY = "UPDATE Employee SET Name = ?, Address = ?, Department = ? WHERE ID = ?";
+    private final String DELETE_QUERY = "DELETE FROM Employee WHERE ID = ?";
+
+    // Create the Employee
+    public int save(Employee employee) {
```

Comment:
Please handle exception for these method

Reply:
Thank you for the feedback, i have finished adding the handle exception for method in the DAO file, (assignment 2 & 3 - Lecture 8), you can checn in the latest commit, thanks.